app/mailboxes/support_mailbox.rb

```diff
@@ -7,6 +7,9 @@ class SupportMailbox < ApplicationMailbox
                     :decorate_mail
 
   def process
+    # prevent loop from chatwoot notification emails
+    return if notification_email_from_chatwoot?
```

Comment:
We can just add it in `before_processing`, as it will stop the cycle of finding the channel and loading the account.

Reply:
@tejaswinichile I am using the decorated mail object inside the helper method to identify the `original_sender` of the email. Since the callbacks are interdependent. let's take it as refactoring for later.